stopes/pipelines/asr_bleu/compute_asr_bleu.py

```diff
@@ -0,0 +1,60 @@
+import asyncio
+import logging
+
+from stopes.pipelines.asr_bleu.configs import AsrBleuConfig
+from stopes.pipelines.asr_bleu.utils import retrieve_asr_config
+from stopes.pipelines.asr_bleu.transcribe_audio import transcribe_audio
+from stopes.pipelines.asr_bleu.retrieve_data import retrieve_data
+
+import hydra
+import sacrebleu
+
+class AsrBleu:
+    def __init__(self, config: AsrBleuConfig):
+        self.config = config
+        self.launcher = hydra.utils.instantiate(self.config.launcher)
+        self.logger = logging.getLogger("asr_bleu")
```

Comment:
`import stopes.core.utils`, and `utils.ensure_dir(self.config.output_dir)` here.